packages/debugger-extension/src/index.ts

```diff
@@ -695,13 +696,34 @@ const main: JupyterFrontEndPlugin<void> = {
       }
     });
 
+    commands.addCommand(CommandIDs.pause, {
+      label: trans.__('Enable / Disable pausing on exceptions'),
```

Comment:
You could improve it by using the status:
```suggestion
      label: service.isPausingOnExceptions ? trans.__('Disable pausing on exceptions') : trans.__('Enable pausing on exceptions'),
```